workflows/colabfold.nf

```diff
@@ -91,7 +91,7 @@ workflow COLABFOLD {
         //
         // MODULE: Run mmseqs
         //
-        if (params.colabfold_model_preset != 'AlphaFold2-ptm') {
+        if (params.colabfold_model_preset != 'alphaFold2-ptm') {
```

Comment:
Should we also add here:
&& params.colabfold_model_preset != 'alphafold2'
as in the webserver mode? Or is it different for some reason?
Which cases does the module MULTIFASTA_TO_CSV need to run for?

Reply:
I really thought it was different but discussing with @athbaltzis it seems to be just a bug since `alphafold2` is not used so often, now is fixed. Good spot! 😄